.github/workflows/basic.yml

```diff
@@ -42,3 +42,16 @@ jobs:
         ${{ matrix.command }}
       env:
         REFERENCE_BRANCH: ${{ github['base_ref'] || github['head_ref'] }}
+
+  publish-pypi:
+    runs-on: ubuntu-latest
+    name: Publish package to PyPI
+    needs: [basic]
+    steps:
+      - name: Publish package
+        if: github.event_name == 'push' && startsWith(github.event.ref, 'refs/tags')
+        uses: pypa/gh-action-pypi-publish@master
+        with:
+          user: __token__
+          password: ${{ secrets.pypi_password }}
+          repository_url: https://test.pypi.org/legacy/
```

Comment:
Nothing about this is Python version specific. The `sdist` can be used by (almost) any version.

And temporary in what sense? ` test.pypi` should be replaced with `pypi` but otherwise it just run on tags and uploads an `sdist`.

Reply:
I meant to test also on python 3.8 (there's only 2.7 and 3.6 above).